randovania/games/samus_returns/gui/dialog/game_export_dialog.py

```diff
@@ -114,12 +127,25 @@ def __init__(self, options: Options, patch_data: dict, word_hash: str, spoiler:
         self.input_file_edit.textChanged.connect(self._on_input_file_change)
         self.input_file_button.clicked.connect(self._on_input_file_button)
 
+        # Input exheader
+        self.input_exheader_edit.textChanged.connect(self._on_input_exheader_change)
+        self.input_exheader_button.clicked.connect(self._on_input_exheader_button)
+
         # Target Platform
         if per_game.target_platform == MSRModPlatform.LUMA:
             self.luma_radio.setChecked(True)
         else:
             self.citra_radio.setChecked(True)
 
+        # Check for auto tracker and/or multiworld
+        if patch_data.get("enable_remote_lua", False):
+            self.citra_radio.setChecked(True)
+            self.luma_radio.setDisabled(True)
```

Comment:
Would be nice to add some text on screen to indicate why it's disabled.
(I would say tooltip, but no one reads these)